File: smartCity/src/main/frontend/src/app/myprofile/image.model.ts
export class ImageModel {
  private _id: number;
  private _name: string;
  private _type: string;
  private _blob:Blob;
  private _uploader:string;


  constructor(name: string, type: string, blob: Blob, uploader: string) {
    this._name = name;
    this._type = type;
    this._blob = blob;
    this._uploader = uploader;
  }

  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get name(): string {
    return this._name;
  }

  set name(value: string) {
    this._name = value;
  }

  get type(): string {
    return this._type;
  }

  set type(value: string) {
    this._type = value;
  }


  get blob(): Blob {
    return this._blob;
  }

  set blob(value: Blob) {
    this._blob = value;
  }


  get uploader(): string {
    return this._uploader;
  }

  set uploader(value: string) {
    this._uploader = value;
  }
}

File: smartCity/src/main/java/com/smartCity/smartCity/Entities/ImageModel.java
package com.smartCity.smartCity.Entities;



import javax.persistence.*;
import java.sql.Blob;
import java.util.Arrays;


@Entity
@Table(name="image_model")
public class ImageModel {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "id")
    private Long id;

    @Column(name = "name")
    private String name;

    @Column(name = "type")
    private String type;

    @Lob
    @Column(name="pic")
    private byte[] pic;

    @Column(name = "uploader")
    private String uploader;

    public ImageModel(){}

    public ImageModel(String name, String type, byte[] pic, String uploader) {
        this.name = name;
        this.type = type;
        this.pic = pic;
        this.uploader = uploader;
    }

    public Long getId(){
        return this.id;
    }

    public void setId(Long id){
        this.id = id;
    }

    public String getName(){
        return this.name;
    }

    public void setName(String name){
        this.name = name;
    }

    public String getType(){
        return this.type;
    }

    public void setType(String type){
        this.type = type;
    }

    public byte[] getPic() {
        return pic;
    }

    public String getUploader() {
        return uploader;
    }

    public void setUploader(String uploader) {
        this.uploader = uploader;
    }

    public void setPic(byte[] pic) {
        this.pic = pic;
    }

    @Override
    public String toString() {
        return "ImageModel{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", type='" + type + '\'' +
                ", pic=" + Arrays.toString(pic) +
                ", uploader='" + uploader + '\'' +
                '}';
    }
}